intent_msg[]  intent_person
int64 total_models
